src/node.cc

```diff
@@ -1148,7 +1149,15 @@ void TearDownOncePerProcess() {
 }
 
 int Start(int argc, char** argv) {
-  InitializationResult result = InitializeOncePerProcess(argc, argv);
+  node::single_binary::NewArgs* newArgs =
+     node::single_binary::checkForSingleBinary(argc, argv);
```

Comment:
“single binary” doesn’t seem to be a particularly good name for this concept?

Also, please read and adhere to https://github.com/nodejs/node/blob/master/doc/contributing/cpp-style-guide.md (`snake_case` for variables and members, `UpperCamelCase` for functions and methods, smart pointers where possible, no redundant `struct` as part of the type name, fix the obvious memory leaks, etc.).

Reply:
Yeah, that’s already much better imo